cloud/ec2_fleet.go

```diff
@@ -174,7 +174,8 @@ func (m *ec2FleetManager) ModifyHost(context.Context, *host.Host, host.HostModif
 	return errors.New("can't modify instances for EC2 fleet provider")
 }
 
-func (m *ec2FleetManager) GetInstanceStatuses(ctx context.Context, hosts []host.Host) ([]CloudStatus, error) {
+// kim; TODO: add test for instance not found.
```

Comment:
Is this still TODO?

Reply:
Nope, I added a test. Removed this comment.